packages/block-library/src/social-links/edit.js

```diff
@@ -75,7 +94,13 @@ export function SocialLinksEdit( props ) {
 		} else {
 			setAttributes( { ...backgroundBackupRef.current } );
 		}
-	}, [ logosOnly ] );
+	}, [
+		customIconBackgroundColor,
+		iconBackgroundColor,
+		iconBackgroundColorValue,
+		logosOnly,
+		setAttributes,
+	] );
```

Comment:
While I'm always looking to fix ESLint errors, there might be unexpected side effects. Let's fix it separately and focus where on the new appender.

Reply:
https://github.com/WordPress/gutenberg/issues/64896

Reply:
Thanks, Dave!